yarn.lock

```diff
@@ -19,21 +19,34 @@ acorn-dynamic-import@^2.0.0:
   dependencies:
```

Comment:
Like discussed I probably had an newer version of yarn and upgraded to the latest version